hexrd/ui/hexrd_config.py

```diff
@@ -413,8 +419,6 @@ def state_to_persist(self):
         for name, _ in self._attributes_to_persist():
             state[self._attribute_to_settings_key(name)] = getattr(self, name)
 
-        state['euler_angle_convention'] = self.euler_angle_convention
```

Comment:
I am deleting this because it seems repetitious. It should already be in the state.